include/rocksdb/db.h

```diff
@@ -1168,6 +1176,14 @@ class DB {
                               uint64_t* manifest_file_size,
                               bool flush_memtable = true) = 0;
 
+  // Retrieve the list of all files in the database.
+  virtual Status GetLiveFilesQuick(std::vector<std::string>& /*ret*/,
+                                   uint64_t* /*manifest_file_size*/,
+                                   SequenceNumber* /*last_sequence*/,
+                                   uint64_t* /*last_decree*/) const {
```

Comment:
I think we can format the code here

Reply:
Here code style is keep the same with L1175, format like what?

Reply:
In pr 23, these code is like this:
```
  // Retrieve the list of all files in the database.
  virtual Status GetLiveFilesQuick(std::vector<std::string>& ret,
                                   uint64_t* manifest_file_size,
                                   SequenceNumber* last_sequence,
                                   uint64_t* last_decree) const { return Status::NotSupported(); }
```
so I guess these code is not format in this pr

Reply:
RocksDB team doesn't emphasize code formatting in their review. So will we do.